index.php

```diff
@@ -0,0 +1,319 @@
+<?php
+
+namespace VUMC\REDCapInstrumentAndVariableSQLRenamer;
+
+$pid = (int)$_GET['pid'];
+if (!array_key_exists("U", $_REQUEST) && $_REQUEST['message'] != "U") {
+    $_SESSION['message'] = "";
+    $_SESSION['message_type'] = "";
+}
```

Comment:
Consider using the symfony http foundation component for the session management and paramater bag: https://symfony.com/doc/current/components/http_foundation.html#accessing-the-session